language-plutus-core/src/Language/PlutusCore/CBOR.hs

```diff
@@ -127,6 +128,7 @@ instance Serialise BuiltinName where
               go 18 = pure ModInteger
               go 19 = pure LtByteString
               go 20 = pure GtByteString
+              go 21 = pure IfThenElse
               go _  = fail "Failed to decode BuiltinName"
```

Comment:
I'd forgotten that this stuff still had to go in the CBOR.  This raises the possibility of two different operators extending the builtins and using the same CBOR tags for different things.  I guess the response to that is "Don't run validators from other blockchains".

Reply:
Note that integers 0-23 get encoded as a single byte but beyond that you require two bytes.  If we go above 23 here, it shouldn't be representing anything that gets used a lot: the most common operations should go at the start.

Reply:
In the Glorious Future these will just become extensible builtins and this will go away. Of course, that means "don't run validators compiled with different sets of builtins", but that seems fair enough.